src/turbtend.F

```diff
@@ -2938,10 +2938,10 @@ subroutine turbw(dt,xh,rxh,arh1,arh2,uh,xf,vh,mh,mf,rho,rf,gz,rgzu,rgzv,rds,sigm
         enddo
         enddo
 
-        !$acc parallel loop gang vector default(present) private(i,j)
+        !$acc parallel default(present)
```

Comment:
There is a dependency of `k` for `dum3` calculation, so the `k` loop must be computed sequentially. I think we need to move the `!$acc parallel default(present)` directive down to the `i,j` loop. Or we could move the `k` loop inside the `i,j` loop and run it sequentially. What do you think? @johnmauff

Reply:
@sjsprecious, I think the current setting for this loop will run the k loop in serial, however, I am not 100% certain of this.  Note that the previous version contains two 'parallel loop' clauses which were not supported by CCE and are not correct due to the k dependency.  Maybe this needs further discussion.

Reply:
My guess is that the `k` loop will be run redundantly in the current setting. Hopefully it is just a matter of efficiency not correctness. But thanks for finding this issue out.

Reply:
@sjsprecious Inverting the loop would potentially have an impact on CPU performance.  Also, a significant change like that might also impact correctness.  The goal of this PR was just to fix the syntax errors that were identified.

Reply:
I see and it makes sense.